### PR TITLE
Add README of English version

### PR DESCRIPTION
**Pull request by Enchan1207**
Excuse me for the sudden pull request :bowing_man:
I was excited GitHub:octocat: was introduced in your video!

I'm rooting for you, lambda-san!

### CODE REVIEW

**Review by rossy0213 — APPROVED** [review]
LGTM

@yoidea 
Could you merge this PR to master?
Looks like I don't have the permisson.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-- 人参を切る
-- 大根を切る
-- 人参と大根に塩をまぶす
-- 人参と大根の水気を切る
-- 砂糖、酢、白だしを測る
-- 大根と人参を液に漬ける
+## Workflow
+
+ - Cut carrots
+ - Cut the radish
```
Comment (yoidea) [inline README.md]: I think it's better to unify the singular or plural.
I wonder "radishes" is good because "carrots" is plural.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+ - Sprinkle salt on carrots and radishes
+ - Drain the carrots and radishes
+ - Measure sugar, vinegar, and white broth
+ - Soak radish and carrot in liquid
```
Comment (yoidea) [inline README.md]: Same as above, I think it is better to unify the singular or plural.

**Review by yoidea — CHANGES_REQUESTED** [review]
Thanks for your contribution👍
I added some comments, please check it.

File: README.md (hunk 1 above)
Comment (rossy0213) [reply README.md]: sounds good 👍

**Comment by Enchan1207** [comment]
Oh, sorry for my poor translation

In accordance with the reviews, I unified some words to plural:

`radish`  -> `radishes`
`carrot` -> `carrots`

**Review by yoidea — APPROVED** [review]

**Comment by yoidea** [comment]
Merged, thanks.

**Pull request merged**